### PR TITLE
enable previously leftout test_usetex

### PR DESCRIPTION
**Pull request by jenshnielsen**
Follow up to #6917 with the tests only on master

### CODE REVIEW

**Comment by jenshnielsen** [comment]
Have not had the time to investigate why the png test fails. It passes with that image for me locally.

**Comment by jenshnielsen** [comment]
Reproduced the failure locally on a Ubuntu 12.04 vm. The text looks correct but is slightly different. I don't think this is a font issue so it is probably due to different tex versions.
Attached diff between travis and current reference from my OSX machine
test_usetex-failed-diff

**Comment by jenshnielsen** [comment]
This still need a skip if not latex decorator

**Comment by jenshnielsen** [comment]
I think this is ready to land now

**Comment by Kojoley** [comment]
@jenshnielsen please try to rearrange extension order. If it will pass with `['png', 'pdf']` then this is #6899 bug.

**Comment by jenshnielsen** [comment]
@Kojoley Good point, I will give it a try

File: lib/matplotlib/tests/test_usetex.py [hunk 1]
```diff
@@ -3,13 +3,20 @@
...
 @image_comparison(baseline_images=['test_usetex'],
                   extensions=['pdf', 'png'],
                   tol=0.3)
 def test_usetex():
+    cmd = ['latex', '-version']
+    try:
+        check_output(cmd)
+    except:
+        from nose import SkipTest
+        raise SkipTest('missing command: %s' % cmd[0])
```
Comment (Kojoley) [inline lib/matplotlib/tests/test_usetex.py]: Why not to use [`skip_if_command_unavailable`](https://github.com/matplotlib/matplotlib/blob/master/lib/matplotlib/testing/decorators.py#L441) decorator?

Comment (jenshnielsen) [inline lib/matplotlib/tests/test_usetex.py]: Please read the commit message

Comment (Kojoley) [inline lib/matplotlib/tests/test_usetex.py]: > Please read the commit message

The actual problem commit is gone there and only comment left. :(

At least you can simplify this with `matplotlib.checkdep_tex()` call.

**Comment by jenshnielsen** [comment]
I think this is ready to merge, the change in coverage seems to be a fluke. The tolerance does not seem to depend on the order it is passing and failing on different backends because of tex version differences. I can reproduce the image that matches travis on a ubuntu 12.04 vm (same os and texlive version as travis) but get different results with a modern texlive install.

**Pull request merged**